### PR TITLE
EZ-393 Integrating map, embed, social media and reference cards compo…

### PR DESCRIPTION
**Pull request by surbhi90**
…nents

### CODE REVIEW

**Comment by kuldev** [comment]
@vaibhavjain-in this looks fine.

File: templates/block/block--inline-block--social-media.html.twig [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+<section{{ attributes.addClass(classes) }}>
+  {{ title_prefix }}
+  {% if label %}
+    <h2{{ title_attributes.addClass('block-title') }}>{{ label }}</h2>
+  {% endif %}
+  {{ title_suffix }}
+
+  {% block content %}
+    {{ content['#contextual_links'] }}
+    {{ kint(content|keys) }}
```
Comment (vaibhavjain-in) [inline templates/block/block--inline-block--social-media.html.twig:76]: Remove Debug statement.

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@surbhi90 Please remove debug statement.

File: templates/block/block--inline-block--social-media.html.twig (hunk 1 above)
Comment (surbhi90) [reply templates/block/block--inline-block--social-media.html.twig]: Removed the Debug statement

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**